packages/core/src/components/SupportButton/SupportButton.tsx

```diff
@@ -54,13 +51,8 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
-export const SupportButton = ({
-  slackChannel = '#backstage',
```

Comment:
Not blocking this at all, agree to remove it, and I might be totally out of the loop with this - But is there some ticket to track the grabbing of metadata from the plugin manifest or package.json? If not we should probably spin up and RFC or something out the back of this so we can track it as a feature request  and come up with a plugin manifest schema or something 👍

Reply:
Haven't seen anything, but makes a lot of sense!  I've seem some of @dtuite 's team customizing the support button contents, for example to send people to Roadie for plugin enhancement, which makes a ton of sense.  So I think an RFC to discuss the level of customization (what is `as config` vs what is just coming through the elements a plugin might pass to the `SupportButton` as `children` elements, etc.)

Reply:
I might be misunderstanding the PR but if it removes the ability to configure the support location on a plugin-by-plugin basis then we are losing some useful functionality.

I asked about Support Buttons before and [Rugvip mentioned](https://github.com/backstage/backstage/issues/2574#issuecomment-700046040) that internal plugins often individually link to a channel for the "owner of the plugin inside the org". 

PS. This internal Backstage screenshot from @katz95 was quite instructive for me personally.

![support button design](https://user-images.githubusercontent.com/61153904/94420756-e721e080-0184-11eb-924e-c2944c971864.png)

Reply:
We definitely want to provide contextual info based on what plugin you are "on" right now, as well. This config data can be used as a fallback to be used mainly where there is no plugin-specific support info, for the "overall app" so to speak.

One tricky thing is, for an open source plugin, what info should one show? One may have a team internally that owns all deployment concerns and provides support for them. If the owners of Backstage internally want to add the Kubernetes plugin, then who supplies what the contact info is for that plugin, and where? Surely it should point to the internal team as far as direct support goes right? Should it still have some open-source-plugin-code-provided links in there for example to the plugin repo where one can directly report bugs in the plugin - or should that still be fielded exclusively by the internal team? Etc. We'll have to settle a system for this.

Reply:
> Surely it should point to the internal team as far as direct support goes right?

Yeah there needs to be a way to contact the internal team for sure.

I think there's a second layer of support that the plugin author may want to provide. For example, I can imagine a company like Snyk (https://github.com/snyk-tech-services/backstage-plugin-snyk) might want to have a docs page on snyk.io which shows end users how to use their plugin. They may also want plugin feature requests to go straight into their GitHub issues rather than going to the internal team who installed the plugin into Backstage but don't control it's features.

Reply:
I agree there probably needs to be more customization options. I only removed the existing links as they were not being used by any plugins, and by the sounds of it wouldn't have fit what we ultimately want to support longer term anyhow.

I'll toss in a different perspective as well. Let's put ourselves in the shoes of a developer working in an org that has adopted backstage. What are the odds they are going to be familiar with the different plugins and or underlying architecture of backstage? Having various support buttons to multiple places for different pages could be confusing and frustrating. For this developer running into an issue, having a single support contact, like the team maintaining the org's backstage instance would be easier. This of course, does put more pressure on that team, but they would be more easily able to route any open source issues to the correct place then an end user imo.

Reply:
I've updated the config structure to support a grouping of `items` and within this contains a list of `links`. This should mimics the functionality in the internal Spotify instance. I also believe the concept of `items` can be easily extended in a future PR so that plugins can add their own `SupportItem` groupings using the same structure.